src/gui/mainwindow.cpp

```diff
@@ -189,11 +189,17 @@ MainWindow::MainWindow(IGUIApplication *app, WindowState initialState)
 
     // Name filter
     m_searchFilter = new LineEdit(this);
-    m_searchFilter->setPlaceholderText(tr("Filter torrent names..."));
+    m_searchFilter->setPlaceholderText(tr("Filter torrents..."));
     m_searchFilter->setFixedWidth(200);
     m_searchFilter->setContextMenuPolicy(Qt::CustomContextMenu);
     connect(m_searchFilter, &QWidget::customContextMenuRequested, this, &MainWindow::showFilterContextMenu);
     m_searchFilterAction = m_ui->toolBar->insertWidget(m_ui->actionLock, m_searchFilter);
+    // Filter by type
+    m_filterBy = new QComboBox(this);
+    m_filterBy->addItem(tr("Torrent names"), TransferListModel::Column::TR_NAME);
+    m_filterBy->addItem(tr("Torrent save paths"), TransferListModel::Column::TR_SAVE_PATH);
+    m_filterBy->setFixedWidth(200);
```

Comment:
What if the combobox items' translations are longer? 
Also i think the "Torrent" part in items is superfluous but that's just my opinion. It's not like there is not enough horizontal space.

Reply:
> Also i think the "Torrent" part in items is superfluous

:+1:
At least in case of "save path".

Reply:
Orignally wanted to put a label like "Filter by " before the combobox and have no "Torrent" in it,
I guess we can also just have no "Torrent" in combobox and assume people will get that its what its filtering by ?

Reply:
>What if the combobox items' translations are longer?

You should remove this line.

Reply:
> You should remove this line.

I'm not shure which line you're taking ? 
I guess you mean there should not be a fixed length

Reply:
>I guess you mean there should not be a fixed length

Yes.